server_src/routes/index.js

```diff
@@ -450,7 +449,7 @@ router.get(
       clientSettings,
       serverSettings,
       systemInformation,
-      db,
+      db: process.env.MONGO,
```

Comment:
the input field should now show the new environment variable, not the config file